src/documents/unity/levelup/Levelup_GettingStarted.html.md

```diff
@@ -68,10 +68,12 @@ Then, you can use the **profile-related _LevelUp_ classes**, such as `SocialLike
 
 ### Caveats
 
-#### iOS Caveats
+#### iOS
 
 1. If you are building your app under Windows, you have to have iTunes installed since the SOOMLA postprocessing is expecting a utility that exists in OS X and is installed with iTunes in Windows.
 
+2. If you are using Google Cardboard SDK in your app, you should remove the `-ObjC` flag from the link flags in Xcode and add `-force_load $(BUILT_PRODUCTS_DIR)/libSoomlaiOSCore.a` and `-force_load $(BUILT_PRODUCTS_DIR)/libSoomlaiOSLevelUp.a` to `Other Linker Flags`. 
+
```

Comment:
@EugeneButusov, 
1. it's more general case not only for Cardboard SDK. 
2. You did not mention bridge libs. Will it work without them?
3. LevelUp depends on Store and Profile, but you did not mention them.